### PR TITLE
General: Update subquery deps and files

### PR DESCRIPTION
**Pull request by Douglasacost**
Setting up a testnet subquery server. What's changed?
- Update docker compose file to the recommended from subquery team
- Update package for deps upgrade
- Change a bit the mapping so is more clear
- Change some type on schema file

Note: Th current deployed version was made using IPFS

### CODE REVIEW

File: nodle-testnet.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        - handler: handleVestingScheduleAdded
+          kind: substrate/EventHandler
+          filter:
+            module: vesting
+            method: VestingScheduleAdded
+        - handler: handleAllocation
+          kind: substrate/EventHandler
+          filter:
+            module: allocations
+            method: newallocation
```
Comment (aliXsed) [inline nodle-testnet.yaml]: Are we going to change this to filter the calls instead and not the event?

File: project.yaml [hunk 2]
```diff
@@ -30,18 +31,13 @@ dataSources:
...
-          filter:
-            module: vesting
-            method: addVestingSchedule
+            method: VestingScheduleAdded
         - handler: handleAllocation
           kind: substrate/EventHandler
           filter:
             module: allocations
-            event: newallocation
+            method: newallocation
```
Comment (aliXsed) [inline project.yaml]: Are we going to change this to filter the calls instead and not the event?

Comment (Douglasacost) [reply project.yaml]: No. Just waiting for te current deploy work to update this PR

File: nodle-testnet.yaml (hunk 1 above)
Comment (Douglasacost) [reply nodle-testnet.yaml]: It's updated now. Tested the current deployed versions  and all is fine

File: project.yaml (hunk 2 above)
Comment (Douglasacost) [reply project.yaml]: It's updated now.

**Review by aliXsed — APPROVED** [review]
Really good job @Douglasacost

**Comment by Douglasacost** [comment]
I'll deprecate this PR this week since a new one is created with better resolutions

**Pull request merged**